other/edge-server/README.md

```diff
@@ -394,21 +417,24 @@ check:
       "bsonType": "string"
     }
   },
-  "required": [
-    "_id",
-    "isComplete",
-    "summary"
-  ]
+  "required": ["_id", "isComplete", "summary"]
 }
 ```
 
 If you've confirmed these details and you still don't see the new `Item`
 that you added in Atlas reflected in the React client, check the App Services
 logs and Edge Server for error messages.
 
-For information about how to check the App Services logs, refer to 
+For information about how to check the App Services logs, refer to
 [View Application Logs](https://www.mongodb.com/docs/atlas/app-services/activity/view-logs/)
 
 ## Issues
 
 Please report issues with the template at: https://github.com/mongodb-university/realm-template-apps/issues/new
+
+## Project npm commands
```

Comment:
I would move this section up and remove the stuff about installing dependencies for the node server and the React client separately. This is the happy path and we want to present it front-and-center.